engine/exchange.go

```diff
@@ -331,16 +331,27 @@ func (bot *Engine) LoadExchange(name string, useWG bool, wg *sync.WaitGroup) err
 	base := exch.GetBase()
 	if base.API.AuthenticatedSupport ||
 		base.API.AuthenticatedWebsocketSupport {
-		err = exch.ValidateCredentials()
-		if err != nil {
+		assetTypes := base.GetAssetTypes()
```

Comment:
You need to check enabled asset types here. eg
```
hasChecked := false
		for k := range base.CurrencyPairs.Pairs {
			if base.CurrencyPairs.Pairs[k].AssetEnabled != nil &&
				*base.CurrencyPairs.Pairs[k].AssetEnabled {
				err = exch.ValidateCredentials(k)
				if err != nil {
					log.Warnf(log.ExchangeSys,
						"%s: Cannot validate credentials, authenticated support has been disabled, Error: %s\n",
						base.Name,
						err)
					base.API.AuthenticatedSupport = false
					base.API.AuthenticatedWebsocketSupport = false
					exchCfg.API.AuthenticatedSupport = false
					exchCfg.API.AuthenticatedWebsocketSupport = false
				}
				hasChecked = true
				break
			}
		}
		if !hasChecked {
			log.Warnf(log.ExchangeSys,
				"%s: Cannot validate credentials, no assetType enabled, Error:\n",
				base.Name)
			base.API.AuthenticatedSupport = false
			base.API.AuthenticatedWebsocketSupport = false
			exchCfg.API.AuthenticatedSupport = false
			exchCfg.API.AuthenticatedWebsocketSupport = false
		} 
```

Its not exactly neat, but it demonstrates what I'm after